proto/elys/amm/query.proto

```diff
@@ -93,26 +93,35 @@ service Query {
     option (google.api.http).get =
         "/elys-network/elys/amm/swap_estimation_by_denom";
   }
+
+  // Queries saved value for a pool and date
+  rpc SavedValue(QuerySavedValueRequest) returns (QuerySavedValueResponse) {
```

Comment:
Query name is not very clear, saved value of what?
Something like may be, `AvgWeightAndSlippageFee` ?? or anything else which makes it clear

Reply:
Changed to `WeightAndSlippageFee`